geode-client-native/src/cppcache/impl/ThinClientRedundancyManager.hpp

```diff
@@ -56,7 +56,7 @@ namespace gemfire {
    void netDown( );
    void acquireRedundancyLock( ) { m_redundantEndpointsLock.acquire_read( ); }
    void releaseRedundancyLock( ) { m_redundantEndpointsLock.release( ); }
-   volatile bool allEndPointDiscon() { return m_IsAllEpDisCon;}
+   bool allEndPointDiscon() { return m_IsAllEpDisCon;} //Copyright: Amdocs Software Systems Limited, 2016
```

Comment:
Source code attribution comments are generally discouraged within ASF projects since the code is owned and maintained by the community, not individual contributors or companies.  

Some references:
http://s.apache.org/iZl

http://mail-archives.apache.org/mod_mbox/incubator-nifi-dev/201505.mbox/%3CCAGHyZ6LcrZ1aPMUjJ%3DrJkkK19Cmveew7PmTUKuWprK_bz3jxmg%40mail.gmail.com%3E